develop/blockchains/maintenance/runtime-metrics-monitoring.md

```diff
@@ -1,4 +1,111 @@
+- Network ID - displays the public network identifier for the node
+- Peer count - indicates the number of peers connected to the node
+- Transactions in queue - shows the number of transactions waiting in the [Ready queue](https://paritytech.github.io/polkadot-sdk/master/sc_transaction_pool_api/enum.TransactionStatus.html#variant.Ready){target=\_blank} for a block author
+- Upload bandwidth - graphs the node's recent upload activity in MB/s
+- Download bandwidth - graphs the node's recent download activity in MB/s
+- State cache size - graphs the size of the node's state cache in MB
+- Block - displays the current best block number to ensure synchronization with peers
+- Block hash - shows the block hash for the current best block number
+- Finalized block - displays the most recently finalized block number to ensure synchronization with peers
+- Finalized block hash - shows the block hash for the most recently finalized block
+- Block time - indicates the time between block executions
+- Block propagation time - displays the time it took to import the most recent block
+- Last block time - shows the time it took to author the most recent block
+- Node uptime - indicates the number of days the node has been online without restarting
+
+## Displaying Network-wide Statistics
+
+In addition to the details available for individual nodes, you can view statistics that provide insights into the broader network. The network statistics provide detailed information about the hardware and software configurations of the nodes in the network, including:
+
+- Software version
+- Operating system
+- CPU architecture and model
+- Number of physical CPU cores
+- Total memory
+- Whether the node is a virtual machine
+- Linux distribution and kernel version
+- CPU and memory speed
+- Disk speed
+
+## Customizing Monitoring Tools
+
+The default telemetry dashboard offers core metrics without additional setup. However, many projects prefer custom telemetry setups with more advanced monitoring and alerting policies.
+
+Typically, setting up a custom telemetry solution involves establishing monitoring and alerting policies for both on-chain events and individual node operations. This allows for more tailored monitoring and reporting compared to the default telemetry setup.
+
+### On-Chain Activity
+
+You can monitor specific on-chain events like transactions from certain addresses or changes in the validator set. Connecting to RPC nodes allows tracking for delays or specific event timings. Running your own RPC servers is recommended for reliable queries, as public RPC nodes may occasionally be unreliable.
+
+### Monitoring Node Operations
+
+Each node can be monitored for basic metrics, such as block height, peer connections, CPU usage, and memory. Substrate nodes expose these metrics at the `host:9615/metrics` endpoint, accessible locally by default. To expose metrics on all interfaces, start a node with the `--prometheus-external` flag.
+
+## Monitoring Tools
```

Comment:
This section could be moved right after the introduction, so readers can understand better the other diagrams that mention some of these tools